vividus-to-zephyr-exporter/build.gradle

```diff
@@ -0,0 +1,33 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath(group: 'org.springframework.boot', name: 'spring-boot-gradle-plugin', version: '2.3.5.RELEASE')
```

Comment:
`2.4.0` now: https://github.com/vividus-framework/vividus/blob/master/vividus-to-xray-exporter/build.gradle#L6